x/bank/keeper.go

```diff
@@ -139,29 +118,27 @@ type ViewKeeper interface {
 	HasCoins(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coins) bool
 }
 
-var _ ViewKeeper = (*BaseViewKeeper)(nil)
-
 // BaseViewKeeper implements a read only keeper implementation of ViewKeeper.
 type BaseViewKeeper struct {
-	am auth.AccountKeeper
+	ak auth.AccountKeeper
 }
 
 // NewBaseViewKeeper returns a new BaseViewKeeper.
-func NewBaseViewKeeper(am auth.AccountKeeper) BaseViewKeeper {
-	return BaseViewKeeper{am: am}
+func NewBaseViewKeeper(ak auth.AccountKeeper) BaseViewKeeper {
+	return BaseViewKeeper{ak: ak}
 }
 
 // GetCoins returns the coins at the addr.
 func (keeper BaseViewKeeper) GetCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins {
-	return getCoins(ctx, keeper.am, addr)
+	return getCoins(ctx, keeper.ak, addr)
 }
 
 // HasCoins returns whether or not an account has at least amt coins.
 func (keeper BaseViewKeeper) HasCoins(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coins) bool {
-	return hasCoins(ctx, keeper.am, addr, amt)
+	return hasCoins(ctx, keeper.ak, addr, amt)
 }
 
-//______________________________________________________________________________________________
+//-----------------------------------------------------------------------------
```

Comment:
why the dash breaks? (lawl don't matter)

Reply:
Consistency with other separations I've seen in the codebase.

Reply:
it's mix and match at this point